src/test/java/seedu/address/commons/util/ConfigUtilTest.java

```diff
@@ -92,13 +92,22 @@ public void saveConfig_allInOrder_success() throws DataConversionException, IOEx
         //Try writing when the file doesn't exist
         ConfigUtil.saveConfig(original, configFilePath);
         Config readBack = ConfigUtil.readConfig(configFilePath).get();
-        assertEquals(original, readBack);
-
+        assertEquals(original.getUserPrefsFilePath().toAbsolutePath(),
+                readBack.getUserPrefsFilePath().toAbsolutePath());
+        System.out.println(original);
+        System.out.println(readBack);
+        System.out.println(original.getUserPrefsFilePath().toAbsolutePath());
```

Comment:
as discussed, to test the absolute vs dynamic path, copy .jar file into empty folder and see whether the window size is saved

Reply:
if its saved, i think we r quite safe